saleor/invoice/error_codes.py

```diff
@@ -0,0 +1,11 @@
+from enum import Enum
+
+
+class InvoiceErrorCode(Enum):
+    REQUIRED = "required"
+    NOT_READY = "not_ready"
+    URL_NOT_SET = "url_not_set"
+    EMAIL_NOT_SET = "email_not_set"
+    NUMBER_NOT_SET = "number_not_set"
+    NOT_FOUND = "not_found"
```

Comment:
Is this code used anywhere?

Reply:
@orzechdev it is, it pops out when trying to trigger invoice mutation with invalid ids. Wouldn't worry about `NOT_FOUND` on frontend-side since that shouldn't occur while normal application use - I've added that to make the backend side more bulletproof.